File: LBV6Library/Models/UserStats.cs
namespace LBV6Library.Models
{
    public class UserStats
    {
        public int TotalUsers { get; set; }
        public int EnabledUsers { get; set; }
        public int SuspendedUsers { get; set; }
        public int BannedUsers { get; set; }
        /// <summary>
        /// How many users have their email address confirmed.
        /// </summary>
        public int ConfirmedUsers { get; set; }
        public int FacebookLogins { get; set; }
        public int GoogleLogins { get; set; }
    }
}

File: LBV6/Scripts/Admin/Models/UserStats.js
function UserStatsModel(data) {
    var self = this;

    self.TotalUsers = data.TotalUsers;
    self.EnabledUsers = data.TotalUsers;
    self.SuspendedUsers = data.SuspendedUsers;
    self.BannedUsers = data.BannedUsers;
    self.ConfirmedUsers = data.ConfirmedUsers;
    self.FacebookLogins = data.FacebookLogins;
    self.GoogleLogins = data.GoogleLogins;
}